app/main.py

```diff
@@ -2,0 +1,21 @@
+def copy_file(command: str) -> None:
+    commands = command.split()
+    if (
+        len(commands) < 3
+        or commands[0] != "cp"
+        or commands[1] == commands[2]
+    ):
+        return
+    try:
+        with (
+            open(commands[1], "r") as source,
+            open(commands[2], "a") as target
+        ):
+            for line in source:
+                target.write(line)
```

Comment:
This is a code quality issue. Although the code is correct, it's a good practice to close the files after you finish using them to free up system resources. You can use the `close()` method to close the files.